using-webauthn-mfa-in-command-line.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Using WebAuthn multifactor authentication in command line
+url: /using-webauthn-mfa-in-command-line
+previous: /using-mfa-in-command-line
+next: /using-otp-mfa-in-command-line
+---
+<em class="t-gray">How to use WebAuthn MFA with gem CLI.</em>
+
+Multifactor authentication (MFA) using WebAuthn works by using a
+removable hardware token or touch biometric / facial biometric capabilities built
+into your phone or computer. This is distinct from MFA based on typing or copying
+a code generated by an authentication app or password manager, called OTP. For
+OTP MFA see "[Using OTP multifactor authentication in command line](/ using-otp-mfa-in-command-line)".
+
```

Comment:
I would mention which RubyGem version supports this. I'm guessing it would be 3.4.11 and above https://rubygems.org/gems/rubygems-update/versions